cluster/config-defaults.yaml

```diff
@@ -105,6 +105,11 @@ skipper_redis_pool_timeout: "250ms"
 skipper_redis_read_timeout: "25ms"
 skipper_redis_write_timeout: "25ms"
 
+# skipper routesrv settings
+skipper_routesrv_replicas: 3
+skipper_routesrv_cpu: "100m"
```

Comment:
does 100m make any sense in this case? I think in production clusters cpu requests non-divisible by 1000 can cause hard to detect problems